static/js/background.js

```diff
@@ -1,8 +1,45 @@
+var windowState = 'normal';
+
 chrome.app.runtime.onLaunched.addListener(function() {
+    if (chrome.storage) {
+        // load stored window settings
+        chrome.storage.sync.get('windowState', function(item) {
+            windowState = (item && item.windowState)
+                ? item.windowState
+                : windowState;
+            createWindow(windowState);
+        });
+    } else {
+        createWindow(windowState);
+    }
+});
+
+function createWindow(state) {
     chrome.app.window.create('index.html', {
-        state: "normal",
-        bounds: { 
-    		width: 850, height: 500
-    	}
+        state: state,
+        bounds: {
+            width: 960,
```

Comment:
++ for correcting this.  I specifically remember sitting at my computer and loosely saying "yeah, that looks like a decent size"